lib/matplotlib/category.py

```diff
@@ -88,7 +88,7 @@ class UnitData(object):
     def __init__(self, data):
         """Create mapping between unique categorical values
         and numerical identifier
-        Paramters
+        Parameters
         ---------
```

Comment:
Need to increase the `-` as well; also, can you add a new line above Parameters?